spec/sql_translator_spec.rb

```diff
@@ -23,10 +23,9 @@
   end
 
   it 'encodes attributes not backed by a database column correctly' do
-    if Gem.loaded_specs["activerecord"].version < Gem::Version.new("4.2.1")
+    if ActiveRecord::VERSION::STRING < "4.2.1"
       skip "the attributes API was included in rails starting in 4.2.1"
-    elsif Gem.loaded_specs["activerecord"].version >= Gem::Version.new("4.2.1") &&
-        Gem.loaded_specs["activerecord"].version < Gem::Version.new("5.0.0")
+    elsif ActiveRecord::VERSION::STRING >= "4.2.1" && ActiveRecord::VERSION::STRING < "5.0.0"
```

Comment:
comparing strings for version numbers could cause issues. "4.2.2" `>` "4.2.19". of course given the versions available, what you have here would work fine, but it is not best practice.